### PR TITLE
Voa 2022

### PR DESCRIPTION
**Pull request by max-hmrc**
Merge 2 reference numbers to 1.

### CODE REVIEW

File: app/views/login.scala.html [hunk 1]
```diff
@@ -102,72 +101,40 @@ <h2 id="error-summary-heading">@Messages("label.checkTheForm")</h2>
...
-                        '_label -> Messages("label.postcode"),
-                        '_labelClass -> "form-label-bold",
-                        'class -> "form-control postcode",
-                        'maxlength -> "8",
-                        '_type -> "hidden"
+                    field = theForm("postcode"),
+                    '_label -> Messages("label.postcode"),
+                    '_labelClass -> "form-label",
+                    'class -> "form-control postcode",
+                    '_type -> "hidden"
```
Comment (Stephen-Goddard) [inline app/views/login.scala.html:122]: Should the text input have a maxlength and pattern?

Comment (Stephen-Goddard) [reply app/views/login.scala.html:122]: Same is true for the reference number above.

Comment (max-hmrc) [reply app/views/login.scala.html:122]: Stephen. Dan wanted it removed as HTML5 validation is not recommended by gov design. I also added novalidation to form for same reason. Terry and Dan asked to remove max length too.
Do the scala tests look ok?

Comment (Stephen-Goddard) [reply app/views/login.scala.html:122]: Ok. That fine if Dan and Terry recommend it I will merge.

**Pull request merged**